lib/assets/connection_cell/main.js

```diff
@@ -815,39 +815,24 @@ export function init(ctx, info) {
     props: {
       fields: {
         type: Object,
-        default: {},
+        default: {
+          base_url: "http://localhost:8123"
+        },
       },
     },
 
     template: `
     <div class="row">
       <BaseInput
-        name="schema"
-        label="Schema"
-        type="text"
-        v-model="fields.schema"
-        inputClass="input"
-        :grow
-      />
-      <BaseInput
-        name="hostname"
-        label="Hostname"
+        name="base_url"
+        label="Base Url"
         type="text"
-        v-model="fields.hostname"
+        v-model="fields.base_url"
```

Comment:
I would keep it as hostname, for consistent with the other DBs, and build the URL internally.